http/analysis-service/server.py

```diff
@@ -84,27 +86,6 @@ def initialize_db(db):
     )
     # Create the table
     db['metadata'].create_all(tables=[scheduled_jobs])
-    # TODO: The above does not create the serial column properly in PostgreSQL.
```

Comment:
I know it's bad practice to leave commented-out gunk in code, but this part could be useful to re-create the table if it somehow got deleted. It would be best to either leave the comment around or figure out how to get the `create_all` function to correctly create the serial `id` column.